Fabric.Identity.API/scripts/Install-Identity.ps1

```diff
@@ -130,8 +134,7 @@ if ($fabricInstallerSecret){
 }
 
 # Call the Idpss powershell script
-$configStore = @{Type = "File"; Format = "XML"; Path = "$installConfigPath"}
-.\Install-IdentityProviderSearchService.ps1 -credential $iisUser.Credential -configStore $configStore -noDiscoveryService:$noDiscoveryService -quiet:$quiet
+.\Install-IdentityProviderSearchService.ps1 -credential $iisUser.Credential -configStore $configStore -noDiscoveryService:$noDiscoveryService -quiet
```

Comment:
Looks like the script is prompting for everything in the identity install script if not quiet, and always calling the idpss install as quiet.

Looks reasonable